cfgov/paying_for_college/models/disclosures.py

```diff
@@ -0,0 +1,771 @@
+            'tuition': self.tuition,
+        }
+        for key in sorted(dict_out.keys()):
+            ordered_out[key] = dict_out[key]
+
+        return json.dumps(ordered_out)
+
+    def as_csv(self, csvpath):
+        """Output a CSV representation of a program"""
+        headings = [
+            'ipeds_unit_id',
+            'ope_id',
+            'program_code',
+            'program_name',
+            'program_length',
+            'program_level',
+            'accreditor',
+            'median_salary',
+            'average_time_to_complete',
+            'books_supplies',
+            'campus_name',
+            'cip_code',
+            'completion_rate',
+            'completion_cohort',
+            'completers',
+            'default_rate',
+            'job_placement_rate',
+            'job_placement_note',
+            'mean_student_loan_completers',
+            'median_student_loan_completers',
+            'soc_codes',
+            'total_cost',
+            'tuition_fees',
+            'test'
+        ]
+        with open(csvpath, 'w') as f:
+            writer = csw(f)
+            writer.writerow(headings)
+            writer.writerow([
+                self.institution.school_id,
+                '',
+                self.program_code,
+                self.program_name,
+                self.program_length,
+                self.level,
+                self.accreditor,
+                self.salary,
+                self.time_to_complete,
+                self.books,
+                self.campus,
+                self.cip_code,
+                "{}".format(self.completion_rate),
+                self.completion_cohort,
+                self.completers,
+                "{0}".format(self.default_rate),
+                "{0}".format(self.job_rate),
+                self.job_note,
+                self.mean_student_loan_completers,
+                self.median_student_loan_completers,
+                self.soc_codes,
+                self.total_cost,
+                self.tuition,
+                self.test
+            ])
+
+
+@python_2_unicode_compatible
+class Alias(models.Model):
+    """
+    One of potentially several names for a school
+    """
+    institution = models.ForeignKey(School)
+    alias = models.TextField()
+    is_primary = models.BooleanField(default=False)
+
+    def __str__(self):
+        return "{} (alias for {})".format(self.alias, self.institution)
+
+    class Meta:
+        verbose_name_plural = "Aliases"
+
+
+@python_2_unicode_compatible
+class Nickname(models.Model):
+    """
+    One of potentially several nicknames for a school
+    """
+    institution = models.ForeignKey(School)
+    nickname = models.TextField()
+    is_female = models.BooleanField(default=False)
+
+    def __str__(self):
+        return "{} (nickname for {})".format(
+            self.nickname, self.institution)
+
+    class Meta:
+        ordering = ['nickname']
+
+
+class BAHRate(models.Model):
```

Comment:
Is this BAHRate model needed/used anywhere? It doesn't seem to really be used/exposed [in college-costs](https://github.com/cfpb/college-costs/search?utf8=%E2%9C%93&q=bahrate&type=).

Reply:
Basic Allowance for Housing is not used in the current tool, but it is not yet clear whether it will  come into play for the public tool.